sbin/pre-commit

```diff
@@ -105,6 +114,8 @@ if ! main 1>"$LOG_TXN" 2>&1; then
     cat "$LOG_TXN"
     cat "$LOG_TXN" >&2
     RET_CODE=1
+elif [[ "${LOG_TXN_SIZE}" != "$(stat -c '%s' "${LOG_TXN}")" ]]; then
```

Comment:
Better to just write to a separate file in main, then check that for non-zero size?